### ISSUE TITLE
Vector package needs printers

### ISSUE THREAD

**Issue by moorepants** [sympy/sympy#10070]
The sympy.vector package could use some custom printers to enhance the readability of the objects in different environments. Inspiration and code can be gathered from the sympy.physics.vector package.

**Comment by tyagi221295** [comment]
@moorepants what actually are you trying to say please explain me??

**Comment by asmeurer** [comment]
It needs printers for the various printers in `sympy.printers` to be defined (string printer, pretty printer, latex, etc.).

**Comment by kevalds51** [comment]
can I take it?

**Comment by debugger22** [comment]
Sure! I don't see any open PR for it.

**Comment by kevalds51** [comment]
How exactly do I implement test cases? Should I do it same the way it is done in physics.vector package?

**Comment by moorepants** [comment]
Yes they can be similar to those tests, but you should check out the tests in the printing package too.

**Comment by tokencolour** [comment]
@kevalds51 Do you plan on working on this?

**Comment by kevalds51** [comment]
@tokencolour I'm hospitalized and won't we able to work on it for another 3-4 days. Someone else can take it if it's high priority.

**Comment by czgdp1807** [comment]
@moorepants Is the current master having printers for vectors. Can you please check and provide the link here. Thanks.

**Comment by vprayag2005** [comment]
@asmeurer @moorepants @oscarbenjamin 
Subject: Interest in Working on Vector Package Printers Issue

Hi SymPy Team,

My name is Prayag, and I’m new to open-source contributions, but I’m very excited to learn and contribute to the SymPy project. I came across the issue regarding the need for printers in the Vector package, and it caught my interest. I’d love to work on this, and as a beginner, I’d appreciate any guidance you could provide.

Could you please explain a bit more about the issue and suggest any initial steps I could take to get started? I’d also be grateful for any tips on navigating SymPy’s codebase to find the relevant parts of the code.

Thank you for your time and consideration, and I look forward to contributing to the project!

Best regards,
Prayag
